### DISCUSSION TITLE
3CB87490-9401-415D-BE97-3E32C4D4C4FF

### DISCUSSION THREAD

**Discussion by giscus** [celestiamobile/comments#119] [General]
# 3CB87490-9401-415D-BE97-3E32C4D4C4FF

NGTS-8 is a star of spectral type K, located at a distance from Earth
1416±18.38314 light-years, a hot Jupiter NGTS-8b was discovered around it in 2019, its mass is slightly smaller than Jupiter, the radius is a little larger than Jupiter, only 5.2 million kilometers from the main star, the orbital period of about 2.5 days, its surface temperature reached 1000 degrees Celsius (1345K).

https://celestia.mobi/addon?item=3CB87490-9401-415D-BE97-3E32C4D4C4FF

**Comment by Galaxy-Leaper1** [comment]
Oh my goodness!  When I type, I always like to start a new line，Causing separation of the introduction
